gubernator/README.md

```diff
@@ -29,3 +29,71 @@ For deployment:
         }
     }
 ```
+
+# GCS Layout API
+
+In order to correctly interpret jobs results, in GCS, Gubernator expects that
+any one job directory is laid out in a specific manner, and that job directories
+are laid out in a specific manner relative to each other.
+
+## Job Artifact GCS Layout
+
+For a single build of a job, Gubernator expects the following layout in GCS:
```

Comment:
Maybe copy the line from docs/federated_testing.md -- "Test results for a build are stored on GCS as gs://$PREFIX/$JOB_NAME/$BUILD_NUMBER. Every run should upload started.json, finished.json, and build-log.txt, and can optionally upload JUnit XML to the artifacts/ directory."